### DISCUSSION TITLE
database export format for ASReview

### DISCUSSION THREAD

**Discussion by sj172** [asreview/asreview#1937] [Q&A]
I am currently using ASReview to conduct a systematic review and have some questions regarding the appropriate export format for PubMed results: 

The 'Data format' section of the ASReview website states that no specific export format for Pubmed is directly supported (https://asreview-readthedocs-io.translate.goog/en/latest/data.html?_x_tr_sl=en&_x_tr_tl=es&_x_tr_hl=es&_x_tr_pto=sc). 

![image](https://github.com/user-attachments/assets/711666ba-5d1a-4770-be91-3a70c91929c5)

Given this, I would like to ask about how I can export my results from Pubmed - Is there a specific PubMed export format that could be adapted or converted for compatibility with ASReview? Or, alternatively, would it be easier to consolidate results from all databases into a reference manager and export them from there in a supported format?

If you have any guidance or advice on this I would be very grateful.

**Comment by taleevenhuis** [comment]
I'm using ASReview regularly and I always import PubMed records first in Zotero and then export them as a RIS-file. You can also use any other reference management-system, like EndNote, Mendeley, Refworks etc. Most important point is that you need a reference management-system to convert PubMed records to RIS-format.
Does this answer your question?

**Comment by FelixWdm** [comment]
@sj172  The answer above explains the recommended workflow, there are tools that convert the PubMed format to RIS, but we're not sure if these are reliable, see for instance: https://www.bibtex.com/c/bibtex-to-ris-converter/. Using a reference manager in between has more additional benefits.
